src/auth/screens/auth-profile.screen.js

```diff
@@ -143,6 +147,38 @@ class AuthProfile extends Component {
                 />
               )}
             </SectionList>
+
+            <SectionList title="Sign Out">
+              <ListItem
+                title="Sign Out"
+                titleStyle={styles.logoutTitle}
+                leftIcon={{
+                  name: 'sign-out',
+                  color: colors.grey,
+                  type: 'octicon',
+                }}
+                onPress={() => {
+                  Alert.alert(
```

Comment:
Would like to have this in a separate function like so: `onPress={() => this.signOut}` and the logic can be abstracted within the `signOut` method (like how I've abstracted `getUserBlog` in a separate method for example)